references/classification/train.py

```diff
@@ -359,7 +359,7 @@ def collate_fn(batch):
         evaluate(model, criterion, data_loader_test, device=device)
         if model_ema:
             evaluate(model_ema, criterion, data_loader_test, device=device, log_suffix="EMA")
-        if args.output_dir:
+        if args.output_dir and (epoch + 1) % args.output_freq == 0:
```

Comment:
To be make sure we always save the last epoch as well, we might want to add:

```suggestion
        if args.output_dir and ((epoch + 1) % args.output_freq == 0) or epoch == args.epochs - 1):
```